ui/src/pages/custody/New.tsx

```diff
@@ -10,27 +9,28 @@ import {
 } from '@daml.js/da-marketplace/lib/Marketplace/Custody/Service';
 import { Party } from '@daml/types';
 import { AssetSettlementRule } from '@daml.js/da-marketplace/lib/DA/Finance/Asset/Settlement';
-import { createDropdownProp, ServicePageProps, makeDamlSet } from '../common';
-import FormErrorHandled from '../../components/Form/FormErrorHandled';
-import BackButton from '../../components/Common/BackButton';
-import { Button, Form, Header } from 'semantic-ui-react';
+import { createDropdownProp, makeDamlSet } from '../common';
 import { DropdownItemProps } from 'semantic-ui-react/dist/commonjs/modules/Dropdown/DropdownItem';
-import { IconClose } from '../../icons/icons';
 import { AllocationAccountRule } from '@daml.js/da-marketplace/lib/Marketplace/Rule/AllocationAccount/module';
 import { VerifiedIdentity } from '@daml.js/da-marketplace/lib/Marketplace/Regulator/Model';
 import { CreateEvent } from '@daml/ledger';
-import paths from '../../paths';
+import ModalFormErrorHandled from '../../components/Form/ModalFormErrorHandled';
+import { Form } from 'semantic-ui-react';
+import { Service as CustodyService } from '@daml.js/da-marketplace/lib/Marketplace/Custody/Service/module';
+import FormErrorHandled from '../../components/Form/FormErrorHandled';
 
 enum AccountType {
   REGULAR = 'Regular',
   ALLOCATION = 'Allocation',
 }
 
-const NewComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> = ({
-  history,
-  services,
-}: RouteComponentProps & ServicePageProps<Service>) => {
-  const party = useParty();
+type Props = {
+  party: Party;
+  custodyServices?: Readonly<CreateEvent<CustodyService, any, any>[]> | undefined;
```

Comment:
I think `<CustodyService, unknown, string>` might work? I typically use that in template type declarations

Reply:
That does work...looks like you can also do `CreateEvent<CustodyService>` and drop the other two